Framework/Core/include/Framework/InputSpan.h

```diff
@@ -10,6 +10,8 @@
 #ifndef FRAMEWORK_INPUTSPAN_H
 #define FRAMEWORK_INPUTSPAN_H
 
+#include <functional>
+
```

Comment:
Do you really need functional in the header? It's an heavy beast.

Reply:
Actually i though we need this to use `std::function`, and that it was simply missing. `InputSpan.h` would then have hidden dependencies. To my surprise `std::function` can be used without functional. But I found that `DataRef.h` is missing. Will change this.

Reply:
Did a test with just including `InputSpan.h`, `functional is indeed a hidden dependency, so I would add it.

Reply:
I have now also changed the order of include files in the test to include `InputSpan.h` first and by that check for hidden dependencies.